src/app.js

```diff
@@ -127,6 +126,12 @@ app.on('ready', () => {
 
     const username = store.get('username');
 
+    if (username === '') {
```

Comment:
Could we move this check up a few lines before we set the application state to Loading?

We will also need to create a context menu, otherwise it will never be created. I think the best solution would be to create it with `0`'s as the values for now - in the future it would be nice to do an object-oriented refactor with proper states.